server/lib/stack.coffee

```diff
@@ -96,7 +97,8 @@ module.exports.addApp = (app, callback) ->
         fs.open config('file_stack'), 'w', (err, fd) ->
             length = data.length
             data = JSON.stringify(data, null, 2)
-            fs.write fd, data, 0, length, 0, callback
+            fs.writeSync fd, data, 0, length, 0
+            callback(err)
```

Comment:
@poupotte I'm a bit surprised here by these 5 lines. Why `length` is computed before `data` is serialized to JSON? And why computing `length` in the first place (it's optional for `fs.write`)?

Maybe someting simpler like these 2 lines is better:

```coffee
data = JSON.stringify data, null, 2
fs.writeFile config('file_stack'), data, callback
```